sympy/solvers/polysys.py

```diff
@@ -20,6 +20,24 @@ def solve_poly_system(seq, *gens, **args):
     """
     Solve a system of polynomial equations.
 
+    Parameters
+    ==========
+
+    seq: a list/tuple/set
+        Listing all the equations that are needed to be solved
+    *gens: generators
+        generators of the equations in seq for which we want the
+        solutions
+    **args: Keyword arguments
```

Comment:
I think that stars should be removed.

Reply:
@sylee957 I will make the necessary changes.